antora-docs/modules/ROOT/pages/authoring.adoc

```diff
@@ -0,0 +1,37 @@
+= Policy Authoring
```

Comment:
Let's update the README.md link to this file (or the published one). Under the section "Getting started for policy authors".